src/index.html

```diff
@@ -1,17 +1,190 @@
 <!doctype html>
-<html lang="en">
+  <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="POST"
+        onsubmit="onSubmit()">
+      <fieldset class="fieldset">
+        <legend class="legend">Personal information</legend>
+
+        <label for="" class="enter_field">
+          Surname:
+          <input
+            id=""
+            type="text"
+            name="Surname"
+            autocomplete="off"
+            required>
+        </label>
+
+        <label for="" class="enter_field">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+            required>
+        </label>
+
+        <label for="" class="enter_field">
+          How old are You?
+          <input
+            type="number"
+            id="age"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+          <label for="age"></label>
+        </label>
+
+        <label for="" class="enter_field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date of birth"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="" class="enter_field_terms">
```

Comment:
```suggestion
        <label for="" class="enter_field_terms">
```
In your css file you have `enter_field` class, but you don't have `enter_field_terms`. Why would you use it here?